### PR TITLE
add reviews

### PR DESCRIPTION
**Pull request by wwu3**
Pls review and approve it.  I will merge it to master

### CODE REVIEW

File: api/v1/views/places_reviews.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+from models.user import User
+from models.review import Review
+from models.place import Place
+from models.state import State
+from models.city import City
+from models import storage
+from models.base_model import BaseModel
+from api.v1.views import app_views
+from flask import request, jsonify, abort
+
```
Comment (jennyngu) [inline api/v1/views/places_reviews.py:14]: I think grouping the class imports together would look more clean

File: api/v1/views/places_reviews.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    place = storage.get(Place, place_id)
+    if place is None:
+        abort(404)
+    json_dict = request.get_json()
+    if not json_dict:
+        abort(400, "Not a JSON")
+    if "user_id" not in json_dict:
+        abort(400, "Missing user_id")
+    if 'text' not in json_dict:
+        abort(400, description="Missing text")
```
Comment (jennyngu) [inline api/v1/views/places_reviews.py]: You can remove "description" here and just have the "Missing text" string as you have it above

File: api/v1/views/places_reviews.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+@app_views.route('/places/<place_id>/reviews', methods=['POST'],
+                 strict_slashes=False)
+def create_review(place_id):
+    place = storage.get(Place, place_id)
+    if place is None:
+        abort(404)
+    json_dict = request.get_json()
+    if not json_dict:
+        abort(400, "Not a JSON")
+    if "user_id" not in json_dict:
```
Comment (jennyngu) [inline api/v1/views/places_reviews.py:58]: Here you have double quotations for "user_id" and below you use single quotations for 'text'. Perhaps change both to either double or single for a cleaner look

**Review by jennyngu — COMMENTED** [review]
Hey, Wendy I've added a few comments for you to have a look at

**Comment by wwu3** [comment]
I've made the above changes.

**Pull request merged**